app/com/linkedin/drelephant/exceptions/TonYExceptionFingerprinting.java

```diff
@@ -17,7 +17,9 @@
 
 import com.google.common.base.Strings;
```

Comment:
Is extracting the Azkaban log is only for Tony and not for Spark ? 

Also I think processing of Azkaban log should be optional (should have property) . Have u done analysis of how much more time it add to exception fingerprinting an application  , if we  add AzkabanLogParsing .   Also Azkaban log parsing is only helpful in cases where there is no relevant information on driver . 

So we should have following  
1) Only parse Azkaban logs if we don't have any exceptions from Driver / executors 
2) Also Azkaban log parsing should be optional (with the property, which saying its taxing and will increase run time )

Reply:
For TonY also Azkaban logs will be fetched. The provision for checking Azkaban logs only when no driver or executor logs are found is in place.

Reply:
I am worried about the trade of between Completeness of  ExceptionFingerprinting(parsing Azkaban logs)  vs total response time . Do u have any analysis for same ?

Reply:
Adding log to show the time spent on Azkaban log analysis.